heroku/cli
# refactor(redis-v5): Move command 'redis:keyspace-notifications'

## Description

Moves the `redis:keyspace-notifications` command to CLI, converting to OClif Core and TS.

## Test instructions

You can test by pulling this branch, installing dependencies with `yarn` and building with `yarn build`. Then run the following commands, knowing that if the app doesn't have any Redis add-on, you should see an error and in apps with more than one Redis add-on you should see an error unless you specify the DATABASE arg.

```
./bin/run redis:keyspace-notifications --help                                         # always test that help looks ok.
./bin/run redis:keyspace-notifications -a test_app_without_redis                      # shows error.
./bin/run redis:keyspace-notifications -a test_app_single_redis -c AKE                # sets the notifications config.
./bin/run redis:keyspace-notifications -a test_app_single_redis -c ""                 # resets the notifications config.
./bin/run redis:keyspace-notifications -a test_app_multiple_redis                     # shows error.
./bin/run redis:keyspace-notifications addon-name -a test_app_multiple_redis -c AKE   # sets the notifications config.
```

You can add Heroku Redis add-ons to any test app with `heroku addons:create heroku-redis -a test-app`.

SOC2 Compliance
[GUS Work Item](https://gus.lightning.force.com/a07EE00001axS4gYAE)
